src/components/LoadButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+
+type Props = {
+  loadPosts: () => void;
+};
+
+export const LoadButton: React.FC<Props> = ({ loadPosts }) => {
+  const [title, setTitle] = useState<string>('Load');
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const loading = () => {
```

Comment:
```suggestion
  const loadTodos = () => {
```
має бути дієслово в інфінітиві